pyanaconda/dbus/observer.py

```diff
@@ -0,0 +1,243 @@
+#
+# Observers of remote DBus objects.
+#
+# Copyright (C) 2017  Red Hat, Inc.  All rights reserved.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+from pyanaconda.dbus import DBus
+from pyanaconda.dbus.constants import DBUS_FLAG_NONE
+from pyanaconda.isignal import Signal
+
+from pyanaconda.anaconda_loggers import get_module_logger
+log = get_module_logger(__name__)
+
+__all__ = ["DBusObserverError", "DBusObserver", "DBusServiceObserver", "DBusObjectObserver"]
```

Comment:
Is there a reason to have `DBusObserver` here? It is a base class so it shouldn't be used anywhere else so or os.

Reply:
Fixed.